controllers/sms-gateway.js

```diff
@@ -119,6 +120,7 @@ module.exports = {
       .then(JSON.parse)
       .then(function(request) {
         // Process webapp-terminating messages asynchronously
+        // TODO these promises do nothing!
```

Comment:
This comment is unclear - it looks like these promises make the contained code asynchronous, but is the implication that this can be simplified?